src/main/java/org/jenkinsci/plugins/github_branch_source/ApiRateLimitChecker.java

```diff
@@ -23,16 +23,14 @@
         public RateLimitChecker getChecker(String apiUrl) {
             return new RateLimitCheckerBase() {
                 @Override
-                boolean checkRateLimitImpl(@NonNull GHRateLimit.Record rateLimit, long count) throws InterruptedException {
+                boolean checkRateLimitImpl(@NonNull GHRateLimit.Record rateLimit, long count, long now) throws InterruptedException {
```

Comment:
This signature will allow us to test this method's behavior without timing sensitivity.